### PR TITLE
Add video to GKE getting started

### LINKED ISSUE #899: Add video to GCP deployment guide
Add [From Zero to Kubeflow](https://www.youtube.com/watch?v=AF-WH967_s4) to the [GCP getting started guide](https://www.kubeflow.org/docs/gke/deploy/).

### PR DESCRIPTION
**Pull request by texasmichelle**
Fixes #899

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/907)

### CODE REVIEW

File: content/docs/started/getting-started-gke.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ description = "Get Kubeflow running on Google Cloud Platform (GCP)"
...
 
-Please refer to the [deployment section](/docs/gke/deploy/) of the 
-[Kubeflow GCP documentation](/docs/gke/) for information on setting up your GCP 
-environment and deploying Kubeflow on Kubernetes Engine (GKE).
+For details on setting up your GCP
+environment and deploying Kubeflow on Kubernetes Engine (GKE),
+refer to the [deployment section](/docs/gke/deploy/) of the
+[Kubeflow GCP documentation](/docs/gke/).
+
+Alternatively, view this walkthrough of installing Kubeflow on GCP via
```
Comment (joeliedtke) [inline content/docs/started/getting-started-gke.md]: Suggestion:

Alternately, view the following walk-through which demonstrates how to install Kubeflow on GCP using
 both the UI and CLI. All code used in this video can be found in the
[examples
repository](https://github.com/kubeflow/examples/tree/master/videos/from_zero_to_kubeflow).

**Comment by texasmichelle** [comment]
Great suggestion, @joeliedtke. How does this look??

**Pull request merged**